azerothcore/azerothcore-wotlk
# fix(Core/GameObject): Pumpkin Shrine

The ability to complete the Headless Horseman via the dungeon finder.

I can't find **any** evidence that the daily quest should be available after the event was moved to the dungeon finder.



## Changes Proposed:
-  Add script to Pumpkin Shrine to spawn Headless Horseman
- Remove the hardcoded texts
- Remove the daily quest from Pumkin Shrine

## Issues Addressed:
- Fixes https://github.com/azerothcore/azerothcore-wotlk/issues/13458

## SOURCE:

## Tests Performed:
- Queued for The Headless Horseman using the Dungeon Finder and selected the gossip option from the Pumpkin Shrine

## How to Test the Changes:


1. Enter The Headless Horseman event from the Dungeon Finder
2. Click the Pumpkin Shrine
3. Select the *Call the Headless Horseman* option
4. Check that things he says, both sound and text, work as intended
5. Try wiping to see if he can spawned again
6. Kill him and make sure he can't be spawned again

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
